### DISCUSSION TITLE
Export MS scan charts as vector graphics with m/z labels

### DISCUSSION THREAD

**Discussion by LouisPM** [OpenChrom/openchrom#551] [Q&A] [answered]
In _OpenChrom_ one can inspect mass spectra in the "Scans" > "Scan Chart" window.
_m/z_ labels can be added to a number of most-intense peaks according to the settings found in said window's "Settings" > "Scans" > "Label Intensities" field.
While I manage to export the plot shown in the "Scan Chart" window as a _.svg_ file, this seems to omit any of the labels seen in the window.

Is there a way to export the plot, including _m/z_ labels, as a vector graphics file (_.svg_, _.pdf_, or similar)?

**Comment by eselmeister** [comment]
Unfortunately, that's not possible at the moment. We plan to do it in the future. To enable this, we have integrated vectorgraphics2d into SWTChart. Currently, we are fixing some issues with the creation of PDF documents.

https://github.com/eclipse/swtchart/issues/417

If that is fixed and an official version of SWTChart is released, we will take care of it.

**Comment by Mailaender** [comment, marked as answer]
This was implemented in https://github.com/OpenChrom/openchrom/pull/710.
